rudder-tests/spec/spec_helper.rb

```diff
@@ -53,6 +54,6 @@
 
 
 RSpec.configure do |c|
-  c.output_stream = File.open('serverspec-result.xml', 'w')
+  c.output_stream = File.open(workspace + '/serverspec-result.xml', 'w')
```

Comment:
if workspace is empty this will write to /
can't you put the / at the end of the variable instead ?